sympy/polys/domains/polynomialring.py

```diff
@@ -33,6 +33,9 @@ def __init__(self, domain_or_ring, symbols=None, order=None):
         self.symbols = ring.symbols
         self.domain = ring.domain
 
+        if domain_or_ring.is_Field and domain_or_ring.is_Exact and len(symbols)==1:
```

Comment:
This is too early. `ring` should be first established, and then `ring.domain` can be used. (rings and domains belong to different classes although they may be mathematically isomorphic. Only domains have these attributes.)